endpoint.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Endpoints
+# Sort in order of priority.
+# index [0] in endpoints_api should be locahost, and
+# index [0] inendpoints_pioneer should hosted URL on same node
+endpoints_api=( \
+    "ws://127.0.0.1:9944" \
+    "wss://testnet.joystream.org/athens/rpc/" \
+    "wss://testnet-brasidas.joystream.org/athens/rpc/" \
+    "wss://testnet-toto.joystream.org/athens/rpc/" \
+)
+
+endpoints_pioneer=( \
+    "wss://testnet-rpc.joystream.org/athens/rpc/" \
+    "wss://testnet.joystream.org/athens/rpc/" \
+    "wss://testnet-brasidas.joystream.org/athens/rpc/" \
+    "wss://testnet-toto.joystream.org/athens/rpc/" \
+)
+
+# Find right name of "endpoint file" in pioneer
+build_endpoint=$(find ~/webui/packages/apps/build -name 'main.*.js')
+
+
+# Find endpoint in pioneer
+for i_p in ${!endpoints_pioneer[@]}
+do
+    if grep -q ${endpoints_pioneer[$i_p]} "$build_endpoint"
+    then
+        wss_pioneer="${endpoints_pioneer[$i_p]}"
+        wss_pioneer_index="$i_p"
+        echo $wss_pioneer
+        break
+    else
+        :
+    fi
+done
+
```

Comment:
We should check here if `wss_pioneer_index` was actually set. If its not set it means we didn't find an endpoint url from `endpoints_pioneer` list, and there is no point proceeding (we will not be able to modify the endpoint value in the built .js file)